src/js/tabs/firmware_flasher.js

```diff
@@ -18,6 +18,18 @@ TABS.firmware_flasher.initialize = function (callback) {
     var intel_hex = false; // standard intel hex in string format
     var parsed_hex = false; // parsed raw hex in array format
     var targetConfig; // the Unified Target configuration to be spliced into the configuration
+    var isConfigLocal = false; // Set to true if the user loads one locally
+    var unifiedConfig; // Unified target configuration loaded from the menu, used when throwing out a local config
+
+    // Is there a list of these elsewhere?, used in parseUnifiedBuilds and ... look for unifiedTargets[0]
+    var unifiedTargets = [ 'STM32F411', 'STM32F405', 'STM32F745', 'STM32F7X2' ];
```

Comment:
Not sure we even need a list of Unified Targets worst case we load a non-unified target, discover from the .hex that it does not support custom defaults, then fail.